testing/e2e/e2e_api_test.go

```diff
@@ -732,3 +721,149 @@ func (s *BeaconKitE2ESuite) TestGetValidatorBalancesForGenesis() {
 		s.Require().True(balance.Balance <= 32e9, "Validator balance should not exceed 32 BERA")
 	}
 }
+
+// getValidator gets the validator with the given stateID and optional options.
+func (s *BeaconKitE2ESuite) getValidator(stateID string, options ...map[string]string) (*http.Response, error) {
```

Comment:
q: what's the difference between using this function with an `option` of `id` versus just using `getStateValidator`?

Reply:
`getValidator` tests https://ethereum.github.io/beacon-APIs/?urls.primaryName=v3.0.0#/Beacon/getStateValidators and `getStateValidator` tests https://ethereum.github.io/beacon-APIs/?urls.primaryName=v3.0.0#/Beacon/getStateValidator.
The latter has validator_id has part of the url path. In the former `getValidator ` id is part of query param. The difference is in the way request is formed.